www/front_src/src/Resources/Graph/Performance/index.tsx

```diff
@@ -183,6 +225,34 @@ const PerformanceGraph = ({
     );
   };
 
+  const translate = (direction: TranslationDirection) => {
+    if (isNil(customTimePeriod)) {
+      return;
+    }
+    const timestampToTranslate =
+      (customTimePeriod.end.getTime() - customTimePeriod.start.getTime()) /
+      translationRatio;
+
+    navigateInGraph?.({
+      start: new Date(
+        add(
+          customTimePeriod.start.getTime(),
+          equals(direction, TranslationDirection.backward)
+            ? negate(timestampToTranslate)
+            : timestampToTranslate,
+        ),
+      ),
+      end: new Date(
+        add(
+          customTimePeriod.end.getTime(),
+          equals(direction, TranslationDirection.backward)
+            ? negate(timestampToTranslate)
+            : timestampToTranslate,
```

Comment:
This should be extracted to a function to avoid duplication. It might be easier to read also